airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -119,6 +150,17 @@ def __init__(
         except ValueError:
             logger.setLevel(logging.WARNING)
 
+    def _get_field(self, extra_dict, field_name):
+        prefix = 'extra__wasb__'
+        if field_name.startswith('extra_'):
+            raise ValueError(
+                f"Got prefixed name {field_name}; please remove the '{prefix}' prefix "
+                f"when using this method."
+            )
+        if field_name in extra_dict:
+            return extra_dict[field_name] or None
+        return extra_dict.get(f"{prefix}{field_name}") or None
```

Comment:
we also write like this `extra_dict.get(f"{prefix}{field_name}", None)` , its upto you

Reply:
difference is empty string